security/oidc-client-mutual-tls/src/test/resources/keycloak-realm.json

```diff
@@ -22,8 +22,14 @@
           "value": "test-normal-user"
         }
       ],
+      "clientRoles": {
+        "test-mutual-tls": [
+          "uma_protection"
```

Comment:
Why this change is needed?

Reply:
Since Keycloak 18 (maybe even in Keycloak 17), some server APIs are protected under `UMA_protection` role. 

https://www.keycloak.org/docs/latest/authorization_services/index.html

Reply:
Thank you, put this in the description, please